src/canary-front-end-screenshots/index.js

```diff
@@ -0,0 +1,130 @@
+const {URL} = require('url');
```

Comment:
This file needs some tests written against it. In particular the `extractUrlsFromSitemap`, `parseSitemap` and `fetchAndParseSitemap` functions. The rest are pulled from an AWS blueprint so I'm less worried about wrapping those in tests
I'll pick those up in a follow up PR